lms/static/sass/views/_teams.scss

```diff
@@ -346,6 +346,106 @@
     }
   }
 
+  .team-profile {
+    .discussion-module {
+      width: 60%;
```

Comment:
it would be great to actually echo the same primary / secondary styling and even naming convention that we use further up for the page. (lined 50 or so of this file) 

1.) This would mean we could wrap .discussion-module in something like .page-content-main {} and then .team-profile would be wrapped in .page-content-secondary {}. 

2.) In addition it would be great to reuse the flex-grid rules used above instead of directly defining width: 60% / 30%. 

___


Here is a copy / paste of the styling for the header area to convey how we might echo the use of flex-grid as well as the naming convention. 

.page-header.has-secondary {
 
       .page-header-main {
         display: inline-block;
         width: flex-grid(8,12);
       }
 
       .page-header-secondary {
         display: inline-block;
         width: flex-grid(4,12);
         @include text-align(right);
       }
   }